pages/stats.js

```diff
@@ -2,53 +2,41 @@ import React, { Component } from "react";
 import Paper from "@material-ui/core/Paper";
 import Grid from "@material-ui/core/Grid";
 import PropTypes from "prop-types";
-
+import { css } from "react-emotion";
 import { withI18next } from "../lib/withI18next";
 import Layout from "../components/layout";
-
 import PrChart from "../components/stats/prChart";
 import PrDurationChart from "../components/stats/pr_duration_chart";
-
-import { withStyles } from "@material-ui/core/styles/index";
 import Container from "../components/container";
 
-const styles = theme => ({
-  marginTop: {
-    marginTop: theme.spacing.unit * 3
-  },
-  table: {
-    // width: 1000
-  }
-});
+const marginTop = css`
+  margin-top: 24px;
```

Comment:
we moved some commonly used values over to theme.js (1px off, feel free to change it)

try:
`import { globalTheme } from "../theme";`
and then
`margin-top: ${theme.marginTop};`

Reply:
nice!

Reply:
lol actually `theme.spacing.unit` was 24, I checked it via the browser inspector before fixing the value here. I think some places had a `+1` added to it though? either way, 25 is fine.